docs/modules/integrations/pages/webhooks/create-and-manage-webhooks.adoc

```diff
@@ -0,0 +1,128 @@
+= Create and Manage Webhook

```

Comment:
Change to "Create and manage webhooks"

We use sentence case throughout the site. And, making plural to be consistent.

Reply:
Changed.